pkg/contract/private_key.go

```diff
@@ -22,22 +22,34 @@ func AddPrivateKeyFlagsToCmd(
 	cmd *cobra.Command,
 	privateKeyFlags *PrivateKeyFlags,
 	goal string,
+	privateKeyFlagName string,
+	keyFlagName string,
+	genesisKeyFlagName string,
 ) {
+	if privateKeyFlagName == "" {
```

Comment:
why do we need this logic. Can we have consistent names across all commands using these flags ?

Reply:
there are commands that have a couple of this flag sets. eg token transferrer can receive two different private keys.